src/ui/public/visualize/loader/index.js

```diff
@@ -17,4 +17,5 @@
  * under the License.
  */
 
-export * from './loader';
+export * from './visualize_loader';
+export * from './visualization_loader';
```

Comment:
Since we don't want to make this a public API and rather remove it soon by the pipeline loader, could we perhaps not export this via the `index.js` file, so that we rather import from `ui/visualize/loader/visualization_loader`, and not having it in the same file that users import the visualize loader from, which we consider public?